kornia/losses/depth_smooth.py

```diff
@@ -6,6 +6,71 @@
 # https://github.com/tensorflow/models/blob/master/research/struct2depth/model.py#L625-L641
 
 
+def _gradient_x(img: torch.Tensor) -> torch.Tensor:
```

Comment:
Why not https://github.com/kornia/kornia/blob/master/kornia/filters/sobel.py#L119 ? With possible "diff" kernel, if you want [-1, 0, 1] kernel